src/services/localStorageService.js

```diff
@@ -0,0 +1,34 @@
+export const removeFromHistory = placeId => {
```

Comment:
Basically this is what I've been talking about when first encountered class usage as service - in this case we have service as module with 3 functions being exported. No action i required here, but this is inconsistency in service creation

Reply:
Hmm, seems that in service shouldn't be work with `localStorage`. Let's move it to `utils`